include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1498,6 +1498,12 @@ FEPointEvaluation<n_components, dim, spacedim, Number>::do_reinit()
     n_q_points_scalar / n_lanes_user_interface +
     (n_q_points_scalar % n_lanes_user_interface > 0 ? 1 : 0);
```

Comment:
Not related to this PR, but since you're here, can you instead use
```suggestion
    (n_q_points_scalar + n_lanes_user_interface - 1) / n_lanes_user_interface;
```